sdk/compute/arm-compute/src/operations/operations.ts

```diff
@@ -32,12 +35,12 @@ export class OperationsImpl implements Operations {
   }
 
   /**
-   * Gets a list of compute operations.
+   * List the operations for the provider
    * @param options The options parameters.
    */
   public list(
     options?: OperationsListOptionalParams,
-  ): PagedAsyncIterableIterator<ComputeOperationValue> {
+  ): PagedAsyncIterableIterator<Operation> {
```

Comment:
This change is from referencing the service itself to referencing the common type
swagger from main
```
"responses": {
          "200": {
            "description": "OK",
            "schema": {
              "$ref": "#/definitions/ComputeOperationListResult"
            }
          },
```
generated swagger from typespec
```
          "200": {
            "description": "Azure operation completed successfully.",
            "schema": {
              "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
            }
          },
```